docs/release_process.md

```diff
@@ -247,11 +267,150 @@ git merge stabilization-vX.Y.Z
+
+#### Fork the repository
+You can skip this step if you already did it in the past. Otherwise:
+- Create a fork from https://src.fedoraproject.org/rpms/scap-security-guide
+    - It can be done via Web UI or using the following command:
+```bash
+fedpkg clone scap-security-guide
+cd scap-security-guide
+fedpkg fork
+```
+- If you forked it via Web UI, clone your fork using `fedpkg`. e.g.:
+```
+fedpkg clone --anonymous forks/<your fedora id>/rpms/scap-security-guide
+cd scap-security-guide
+```
+More information about using `fedpkg` anonymously is found in [Fedora Package Tools](https://docs.fedoraproject.org/en-US/package-maintainers/Package_Maintenance_Guide/#using_fedpkg_anonymously).
+
+It is preferred to update the `scap-security-guide.spec` file via PR instead of [committing directly](https://docs.fedoraproject.org/en-US/package-maintainers/Package_Maintenance_Guide/#typical_fedpkg_session),
+even you are the a maintainer of the `scap-security-guide` package.
+This ensures that at least two maintainers are involved in the process and make it much less prone
+to human errors.
+
+#### Update the spec file
+Usually the changes are straightforward in the `scap-security-guide.spec` file.
+It will be necessary to update the `Version:` line with the new version. e.g.:
+```
+Version:	0.1.65
+```
+This should be the only line to be changed in most cases. But your should review the file
+completely and propose improvements if any opportunity is found. Updates regarding [Package Maintenance Guide](https://docs.fedoraproject.org/en-US/package-maintainers/Package_Maintenance_Guide)
+or new features for `spec` files might be welcome.
+
+Once the Version is updated and the `scap-security-guide.spec` file is reviewed, it is necessary
+to append the `%changelog` section. For reference, this was included when the package was rebased
+to `0.1.65`:
+```
+%changelog
+* Tue Dec 06 2022 Marcus Burghardt <maburgha@redhat.com> - 0.1.65-1
+- Update to latest upstream SCAP-Security-Guide-0.1.65 release:
+  https://github.com/ComplianceAsCode/content/releases/tag/v0.1.65
+```
+These changes should be enough for most cases. Save the file and proceed to the next steps.
+
+#### Update the source
+When the `Version:` line is updated in the `scap-security-guide.spec` file, the `Source0` line is
+also impacted.
+- Ensure the sources are downloaded locally:
+```bash
+fedpkg sources
+```
+- Check the new sources and ensure they are correct.
+- To ensure the `scratch build` doesn't fail due to an "Invalid Source", ensure the new sources
+are uploaded to the [lookaside_cache](https://docs.fedoraproject.org/en-US/package-maintainers/Package_Maintenance_Guide/#_upload_new_source_files_to_the_lookaside_cache):
+```bash
+fedpkg new-sources
+```
+- This command will change the `source` and `.gitignore` files. Please, check the changes.
+
+### Package Tests
+- Check if the changes work as expected:
+```bash
+fedpkg mockbuild
```

Comment:
This should be another valid way to do the same. The `mockbuild` is in the Fedora documentation. I think it would be good to mention this alternative way. Would you mind to update the doc, please?

Reply:
I split the build section and added an updates section.

Reply:
Great. Thanks.